### PR TITLE
Add new issue templates

### PR DESCRIPTION
**Pull request by kieranogrady**
- Add hidden github directory for issue templates
- Move existing bug report issue template
- Add new feature request issue template

Issue: #2136

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
Comment (JonRowe) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Can you add a note about discussing on the mailing list first? See above

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```
Comment (JonRowe) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Some of the text below makes the assumption you could be doing something beyond a bug report, fancy tweaking the text a bit?

**Comment by benoittgt** [comment]
Thanks a lot for this contribution. I follow Jon requests.

**Comment by kieranogrady** [comment]
Hi @JonRowe, thank you for your review and feedback. I'm offline the next few day on holiday but I'll push the requested amendments when I return.

**Comment by kieranogrady** [comment]
> Thanks for this, its something thats been on our radar to do, I'd love it if the feature request was slightly more fleshed out though!

Hi, I've updated the notes in proposed issue templates. Please let me know what you think

**Comment by benoittgt** [comment]
Thanks a lot!

**Pull request merged**